queries/dune_v1/period_slippage.sql

```diff
@@ -8,14 +8,21 @@ filtered_trades as (
            solver_name,
            solver_address,
            trader                                                as trader_in,
-           receiver                                              as trader_out,
+           -- Null Receiver: https://dune.com/queries/1729130
+           case
+                when receiver = '\x0000000000000000000000000000000000000000'
+                then trader
+            else receiver end                                    as trader_out,
```

Comment:
> I am not going to separate this change into another PR, so don't bother mentioning it.

But why? If this is a bug that needs to be fixed anyways why include it into some PR that is doing something else? If we want to revert this PR in case it caused problems, we would also revert this perfectly necessary bug fix.

Other than the overhead of 5 minutes extra git magic, is there a reason to not separate these two changes?

Reply:
Well. this was because I would not have had time to fix it before we wanted to do the payout. Now that its already too late, I suppose we can let this sit longer and include the fix here.

Reply:
Here is the relevant PR - https://github.com/duneanalytics/spellbook/pull/2300

We can come back here and fix this once that change is reflected in Dune.